salt/version.py

```diff
@@ -79,7 +79,7 @@ class SaltVersionsInfo(type):
     SILICON       = SaltVersion("Silicon"      , info=3004,       released=True)
     PHOSPHORUS    = SaltVersion("Phosphorus"   , info=3005,       released=True)
     SULFUR        = SaltVersion("Sulfur"       , info=(3006, 0),  released=True)
-    CHLORINE      = SaltVersion("Chlorine"     , info=(3007, 0))
+    CHLORINE      = SaltVersion("Chlorine"     , info=(3007, 0),  released=True)
```

Comment:
False

Reply:
@s0undt3ch 

Good catch, I did this while trying to figure out why the docs builds were failing (on the branch tests). Turns out it was this; 
https://github.com/saltstack/salt/pull/64304/files#diff-984932af3bb250046260d0053343612dbbb7e4b3fd59d9732fccaa8993cb9f50L22
which is needed on the branch tests unless we change something with this:
https://github.com/saltstack/salt/commit/0b0c29e79cf878e1678bd4f9ed4edcd5978ad54b
Not sure what the expectation here should be.. It only matters because I was pushing this work to a branch under the saltstack org.